charts/sck-otel/templates/_pod.tpl

```diff
@@ -6,6 +6,21 @@ imagePullSecrets:
 serviceAccountName: {{ include "splunk-otel-collector.serviceAccountName" . }}
 securityContext:
   {{- toYaml .Values.agent.podSecurityContext | nindent 2 }}
+initContainers:
+  - name: chown
+    image: registry.access.redhat.com/ubi8/ubi
+    securityContext:
+      runAsUser: 0
+    command: ['sh', '-c', "chown -Rv 20000:20000 {{ .Values.checkpointPath }}; chmod -v g+rwx {{ .Values.checkpointPath }}{{ if .Values.containerLogs.enabled }}; setfacl -R -d -m g:20000:rx /var/log;setfacl -R -d -m g:20000:rx /var/lib/docker/containers{{ end }}"]
+    volumeMounts:
+      - name: checkpoint
+        mountPath: {{ .Values.checkpointPath }}    
+      {{- if .Values.containerLogs.enabled }}
+      - name: varlog
+        mountPath: /var/log
+      - name: varlibdockercontainers
+        mountPath: /var/lib/docker/containers
+      {{- end }}
```

Comment:
Is this expected addition? Looks unrelated

Reply:
oh shoot, i was playing with it and i shouldn't have included it. let me revert it.